src/Service.Tests/MultiSourceTestSchema.gql

```diff
@@ -0,0 +1,627 @@
+type Clients @model(name: "Clients") {
+  Index: Long @primaryKey(databaseType: "Int64")
+  CustomerId: String
+  FirstName: String
+  LastName: String
+  Company: String
+  City: String
+  Country: String
+  Phone1: String
+  Phone2: String
+  Email: String
+  SubscriptionDate: DateTime
+  Website: String
+}
+
+"Order by input for Clients GraphQL type"
+input ClientsOrderByInput {
```

Comment:
Even for MultiSourceTestSchema, the customer shouldnt need to provide the Input/Mutation/Query types, only the Base Entity types should be required and that too only for CosmosDB_NoSQL. See e.g. schema.gql

Reply:
Im not using this as input for the cosmosDB_NOSQL db type. rather im using the whole schema for the test case in general. instead of writing integration test where schema would be built first and then we run the query, im using a prepopulated schema so it enables us to unit test only the query execution portion. The other portions are tested in their respective test cases (Builder test cases)